README.md

```diff
@@ -1,73 +1,78 @@
+## Live Demo
 
+Check the live demo for this project [here](https://covid-19-monitor2.netlify.app/).
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
-### Setup
+You need to be able to use **Node Package Manager**, or also called, the `npm` command.
+If you dont have it, install it with these commands in the CLI (for Linux):
 
-### Install
+`curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash -`
 
-### Usage
+`sudo apt install nodejs`
+
+Then, run this command in the CLI:
 
-### Run tests
+`npm --version`
 
-### Deployment
+If the previous command prints a single line with a version number on screen, `npm` was installed.
+### Install
 
+First you need to clone the repository into the folder of your choice with this command:
 
+`git clone https://github.com/Yothu/covid-19-monitor.git`
 
-## Authors
+Then go inside the repository foler with this command:
 
-👤 **Author1**
+`cd covid-19-monitor`
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+After you are inside, download the `npm` dependecies with this command:
 
-👤 **Author2**
+`npm install`
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+Finally, show the project through the browser with this command:
 
-## 🤝 Contributing
+`npm start`
 
-Contributions, issues, and feature requests are welcome!
+### Usage
 
-Feel free to check the [issues page](../../issues/).
++ Once in the browser, you will see the **Home page**, in it you will see two sections, the *country section* and the *regions section*, in the first one you will se the map of the country the covid app is based in, and also its covid statistics. On the *regions section* you will see all the regions of the country listed, you can filter them by today's `confirmed` cases and `deaths`.
++ Once you click on a region you will be redirected to the **details page**, in it there are the covid statistics of that region, showing the region's flag, and statistics.
++ You can press the left arrow in the header to go back to the **Home page**.
 
-## Show your support
+## Author
 
-Give a ⭐️ if you like this project!
+👤 David Vergaray
 
-## Acknowledgments
+- GitHub:   [@Yothu](https://github.com/Yothu)
+- Twiter:   [@Daivhy](https://twitter.com/Daivhy)
+- LinkedIn: [David Vergaray](https://www.linkedin.com/in/david-vergaray-almontes-051a11127/)
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
```

Comment:
As required, Kindly add an "Acknowledgements" section that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). 👍

Reply:
I forgot about it! Thanks for the reminder!